src/test/java/com/thebois/models/inventory/ColonyInventoryTests.java

```diff
@@ -0,0 +1,124 @@
+package com.thebois.models.inventory;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.Stack;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.Mockito;
+
+import com.thebois.models.IFinder;
+import com.thebois.models.beings.Colony;
+import com.thebois.models.beings.pathfinding.IPathFinder;
+import com.thebois.models.inventory.items.IItem;
+import com.thebois.models.inventory.items.ItemType;
+import com.thebois.models.inventory.items.Log;
+import com.thebois.models.inventory.items.Rock;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class ColonyInventoryTests {
```

Comment:
Weren't colony inventory tested before? Or did you just move it?

Reply:
Looks like to me that only the new methods that were added to `IInventory` are tested in here. The rest are probably still tested in `ColonyTests`. Should probably be moved in here as well.

Reply:
Since the method I added wasn't needed, ColonyInventoryTest has now been removed completely